cmd/containerd/command/service_windows.go

```diff
@@ -108,6 +116,20 @@ func applyPlatformFlags(context *cli.Context) {
 		*v.d = context.GlobalBool(v.name)
 	}
 	logFileFlag = context.GlobalString("log-file")
+	priorityClassName = context.GlobalString("priority-class")
```

Comment:
I think we should just save the flag value to `priorityClassName` here, and actually set the priority below where we run the service (in `registerUnregisterService`). This function seems to just be used for parsing flags, rather than using them.

Then we also don't need to make this function return an error.